src/language-js/printer-estree.js

```diff
@@ -3322,8 +3322,35 @@ function shouldGroupFirstArg(args) {
   );
 }
 
+const functionCompositionFunctionNames = {
+  pipe: true,
+  pipeP: true,
+  pipeK: true,
+  compose: true,
+  composeFlipped: true,
+  composeP: true,
+  composeK: true,
+  flow: true,
+  flowRight: true
```

Comment:
Maybe add as comments which is coming from which library, so we know in the future where it came from